packages/opentelemetry-api/src/metrics/BoundInstrument.ts

```diff
@@ -43,3 +44,12 @@ export interface BoundMeasure {
     spanContext: SpanContext
   ): void;
 }
+
+/** An Instrument for Observer Metric. */
```

Comment:
I would prefer: **Base interface for all the Observer metrics.**

Reply:
I'm not sure if I understand this correctly, this interface `BoundObserver` is an interface with just one function `setCallback` and it doesn't inherit from anything. How would you create a base interface from this as this already looks like a "base" ?